--- origsrc/anjuta-2.24.2/scripts/build-schemas.mk	2008-10-19 10:34:47.000000000 -0500
+++ src/anjuta-2.24.2/scripts/build-schemas.mk	2008-12-18 11:43:55.556375000 -0600
@@ -4,7 +4,7 @@
 prefs_glade_schemas_DATA = $(prefs_glade_files:.glade=.schemas)
 
 %.schemas: %.glade
-	$(top_srcdir)/scripts/glade2schema.pl $(srcdir)/$(?) > $(@)
+	$(top_srcdir)/scripts/glade2schema.pl $(?) > $(@)
 
 if GCONF_SCHEMAS_INSTALL
 install-data-local:
