--- origsrc/anjuta-3.18.2/configure.ac	2015-11-15 20:51:20.481199000 -0600
+++ src/anjuta-3.18.2/configure.ac	2015-11-15 20:52:58.415093600 -0600
@@ -329,7 +329,7 @@ if test "$user_disabled_vala" = 1; then
 else
         AC_MSG_RESULT(no)
 	AC_MSG_CHECKING(if libvala is available)
-	m4_foreach([VERSION], [[0.18], [0.20], [0.22], [0.24], [0.26], [0.28]],
+	m4_foreach([VERSION], [[0.18], [0.20], [0.22], [0.24], [0.26], [0.28], [0.30]],
 		[PKG_CHECK_EXISTS([ libvala-VERSION ],
 			 [ valaver="-VERSION" ])
 			 ])
