--- origsrc/anjuta-2.4.2/plugins/glade/Makefile.am	2008-04-06 13:13:14.000000000 -0500
+++ src/anjuta-2.4.2/plugins/glade/Makefile.am	2008-06-04 23:13:53.609375000 -0500
@@ -41,6 +41,7 @@
 # Plugin dependencies
 libanjuta_glade_la_LIBADD = \
 	$(PLUGIN_GLADE_LIBS) \
+	$(GNOME_VFS_LIBS) \
 	$(LIBANJUTA_LIBS)
 endif
 
--- origsrc/anjuta-2.4.2/plugins/profiler/gprof-view.h	2008-04-06 13:13:03.000000000 -0500
+++ src/anjuta-2.4.2/plugins/profiler/gprof-view.h	2008-06-04 23:13:53.609375000 -0500
@@ -26,6 +26,7 @@
 
 #include <glib.h>
 #include <glib-object.h>
+#include <glib/gi18n.h>
 #include <gtk/gtk.h>
 #include <libanjuta/interfaces/ianjuta-symbol-manager.h>
 #include <libanjuta/interfaces/ianjuta-document-manager.h>
--- origsrc/anjuta-2.4.2/scripts/build-schemas.mk	2008-04-06 13:12:53.000000000 -0500
+++ src/anjuta-2.4.2/scripts/build-schemas.mk	2008-06-04 23:45:34.000000000 -0500
@@ -4,7 +4,7 @@
 prefs_glade_schemas_DATA = $(prefs_glade_files:.glade=.schemas)
 
 %.schemas: %.glade
-	$(top_srcdir)/scripts/glade2schema.pl $(srcdir)/$(?) > $(@)
+	$(top_srcdir)/scripts/glade2schema.pl $(?) > $(@)
 
 if GCONF_SCHEMAS_INSTALL
 install-data-local:
